src/services/signUp.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
```

Comment:
you could put the service's functions like login/signup in one file only, no need to put them in different files. Again, just a suggestion